src/hera/workflows/v5/http.py

```diff
@@ -0,0 +1,60 @@
+from typing import List, Optional
+
+from hera.workflows.models import (
+    HTTP as _ModelHTTP,
+    HTTPBodySource,
+    Template as _ModelTemplate,
+    V1HTTPHeader as HTTPHeader,
+)
+from hera.workflows.v5._mixins import _SubNodeMixin, _TemplateMixin
+
+
+class HTTP(_TemplateMixin, _SubNodeMixin):
+    url: str
+    body: Optional[str] = None
+    body_from: Optional[HTTPBodySource] = None
+    headers: Optional[List[HTTPHeader]] = None
+    insecure_skip_verify: Optional[bool] = None
+    method: Optional[str] = None
+    success_condition: Optional[str] = None
+    timeout_seconds: Optional[int] = None
```

Comment:
This class is a good candidate for a discussion topic: are we increasing the maintenance burden on this repo (i.e. Hera devs) by copy/pasting a subset of member variables from the `Template` model class?

Could we instead subclass it (for example), which gives us the Pydantic `Field` info, and reduces the need for us to keep things in sync. If we could subclass `Template`, somehow bring forward the relevant fields for code completion when creating an `HTTP` object, and ensure mutually exclusive fields from the parent `Template` class aren't available, this might be a cleaner solution? Then the `_build` methods could return a copy of `self` with certain fields build (here it would be `http` and `metadata`.

Obviously I haven't fully fleshed out the alternatives - just an idea to bounce off for a discussion (slack or an issue?) 🚀